packages/sitecore-jss-nextjs/src/middleware/personalize-middleware.test.ts

```diff
@@ -7,7 +7,7 @@ import sinon, { spy } from 'sinon';
 import nextjs, { NextRequest, NextResponse } from 'next/server';
 import { debug } from '@sitecore-jss/sitecore-jss';
 import { ExperienceParams } from '@sitecore-jss/sitecore-jss/personalize';
-
+import { SiteResolverType } from '@sitecore-jss/sitecore-jss/site';
```

Comment:
We removed `SiteResolverType`, right? Not sure how this is being imported/used in tests.

Reply:
Right, strange that unit tests are passed